frontend/src/metabase/xray/components/InsightCard.jsx

```diff
@@ -178,14 +178,14 @@ export class VariationTrendInsight extends Component {
 
     return (
       <InsightText>
-        It looks like this data has grown {mode}ly{" "}
+        {t`It looks like this data has grown ${mode}ly`}{" "}
```

Comment:
We'll probably want to add a map like 

```javascript
const MODE_ADVERB_STRINGS = {
  "big": t`bigly`,
  "huge": t`hugely`,
  // ...
}
```

(I don't know what the actual values of `mode` can be. @sbelak?)

Then do:

```javascript
{t`It looks like this data has grown ${MODE_ADVERB_STRINGS[mode]}`}{" "}
```